README.macOS.md

```diff
@@ -0,0 +1,120 @@
+
+## Step 3: Install [MongoDB](https://docs.mongodb.com/manual/tutorial/install-mongodb-on-os-x/)
+MongoDB will be your database for storing any information passed to Homebrewery such as your brews.  This is how brews are stored on the Homebrewery website unless a user decides to use their Google Drive account.
+
+### MacOS Catalina, Big Sur, or newer
+In the terminal: 
+- create a folder to store the Mongo data:  `sudo mkdir -p /System/Volumes/Data/data/db`
+- set the permissions on that folder:  `sudo chown -R `\`id -un\`` /System/Volumes/Data/data/db`
+- use Homebrew to add MongoDB:  `brew tap mongodb/brew`  followed by `brew install mongodb-community@5.0`  (as of Sep 2021)
+- start MongoDB as a background process: `brew services start mongodb-community@5.0`
+
+You should see `Successfully started 'mongodb-community'...`.   MongoDB will always run as a background process going forward, and restart when you restart your computer.  You could change this by substituting `brew services start...` with `brew services run...` which will terminate Mongo when you turn off of your computer (and will need to be run again when you need it).  
+
+### Setting up "Aliases" (optional)
+Aliases allow you to create shortcuts in the terminal, and the below is a totally optional method of creating shortcuts to better manage your Mongo database service.  
+
+- In Finder, navigate to your "home" directory (generally is the folder with your username and a house icon)
+- Reveal hidden system files and folders by hitting `shift` `cmd` `.` and you'll see those items appear, greyed out. 
+- If there is a `.zshrc` file already, open it...if there isn't, open a text editor (TextEdit will work) and create a file just named `.zshrc` and click okay when it warns you that such files are used by the system.
+- add the following lines to the file and save it.
+```
+alias mongod-run="brew services run mongodb-community@5.0"
+alias mongod-start="brew services start mongodb-community@5.0"
+alias mongod-status="brew services list"
+alias mongod-stop="brew services stop mongodb-community@5.0"
+alias mongod-restart="brew services restart mongodb-community@5.0"
+```
+- In the Terminal, run `source ~/.zshrc`
+- Now you should be able to run the shortcuts.  In the Terminal, run `brew services status` and after hitting Enter you should get a list of services running and their status (if you started your MongoDB service it should be listed).
+
+## Step 4: Create Homebrewery Database
+In the Terminal:
+- type `mongosh` and you will get a result indicating you've opened Mongo
+- type `use homebrewery` to create a database for Homebrewery
+- type `db.brews.insert({"title":"test"})` and you will insert a new document titled "test" into the "brews" collection, in the Homebrewery database.
+- type db.brews.find().pretty() to display a list of the documents in the "brews" collection...you should get one result back, the document you just created.
```

Comment:
this should be `db.brews.find().pretty()`  (just put that in a code block).

Reply:
line 54 gets this error:
```zsh
This gets an error about deprecation:
```zsh
homebrewery> db.brews.insert({"title":"test"})
DeprecationWarning: Collection.insert() is deprecated. Use insertOne, insertMany, or bulkWrite.
```